quantum/nodes/quantum-circuit/quantum-circuit.js

```diff
@@ -20,6 +25,32 @@ module.exports = function(RED) {
     const output = new Array(this.outputs);
     const node = this;
 
+    flowContext.set('quantumCircuitReadyEvent', quantumCircuitReady);
```

Comment:
I'm very happy with this solution which is more robust and working better than the initial solution I implemented!
Thank you!

However, I would like to minimize the number of `flow` variables used in order to keep the code simple and error-free: what happens if the user sets a `flow` variable with the same name as the ones we are using? --> We will probably overwrite the user's data...

Ideally, we would have:
1) The same logic without using `flow` variables - Not sure it is possible ??
2) The same logic with a single `flow` variable object that could store all 3 `flow` variables. The name should be very specific to limit the probability of the user choosing the same variable name. Finally, the `flow` variables should be deleted once they are not in use anymore (I found how to make `flow` variables disappear from the editor in `quantumRegister.js` & `classicalRegister.js`).

Reply:
The flow context variable is not cleared after the circuit ready event. An enhancement #47 has been proposed to implement better state management.